donut/modules/calendar/templates/events_list-iframe.html

```diff
@@ -22,7 +22,7 @@
               <label for="name">Event Name:</label>
             </td>
             <td class="searchTable"><input type="text" id="query" name="query" class="form-control" /></td>
-            <td>              <input class="btn" type="submit" value="Submit" /></td>
+            <td>              <input class="btn btn-primary" type="submit" value="Search" /></td>
```

Comment:
Please also change the binding event in /calendar/static/list_script.js -- it's looking for a button with a 'Submit' value right now.